packages/fast-components/src/dialog/dialog.styles.ts

```diff
@@ -0,0 +1,46 @@
+import { css } from "@microsoft/fast-element";
+
+export const DialogStyles = css`
+    :host([hidden]) {
+        display: none;
+    }
+
+    :host {
+        --elevation: 8;
+        --height: 480px;
+        --width: 640px;
```

Comment:
@nicholasrice do you have any concerns about using custom properties here for the dialog?

Reply:
I don't really understand what utility they add

Reply:
nevermind I see now